index.php

```diff
@@ -72,34 +113,27 @@
 
                     <label class="checkbox">
                             <input class="checkbox__input visually-hidden show_completed"
-                            type="checkbox" <?php if ($show_complete_tasks == 1): ?>checked<?php endif; ?> >
+                            type="checkbox" <?php echo $show_complete_tasks == 1 ? 'checked' : '' ?> >
                         <span class="checkbox__text">Показывать выполненные</span>
                     </label>
                 </div>
-
                 <table class="tasks">
-                <tr class="tasks__item task">
+                <?php foreach($tasks as $key => $val): ?>
+                <?php if ($val['completed'] && $show_complete_tasks == 0) : ?>
```

Comment:
Тут будет тогда вот так
```suggestion
                <?php if ($task['completed'] && $show_complete_tasks == 0) : ?>
```